src/test/java/core/basesyntax/separator/FruitTransactionsParserImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.separator;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Transaction;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionsParserImplTest {
+    private static List<String> validTestData;
+    private static List<String> invalidTestData;
+    private static FruitTransactionsParser parser;
+
+    @BeforeClass
+    public static void setUo() {
+        validTestData = List.of(
+                "type,fruit,quantity",
+                "b,apple,100",
+                "b,banana,100");
+        invalidTestData = List.of(
+                "type,fruit,quantity",
+                "b,100,apple");
+        parser = new FruitTransactionsParserImpl();
+    }
+
+    @Test
+    public void create_validTransaction_Ok() {
```

Comment:
```suggestion
    public void transactionsParser_validTransaction_Ok() {
```

There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.